Header header
float64 rollAng
float64 pitAng
float64 headAng
float64 rStaDev
float64 pStaDev
float64 hStaDev